### ISSUE TITLE
Ban Appeal:  Ecking03

### ISSUE THREAD

**Issue by Ecking03** [Kyrobi/Cynagen#212]
Please title this post "Ban Appeal:" followed by a name

**Minecraft username**:

**I was banned on**: (Format: Dec 23, 2012)

**I was banned by**: (Leave blank if unknown)

**I was banned for**:

**A brief summary of what happened before the ban was**:

**I like to be unbanned because**:

**Explain why is cereal is/isn't considered a soup**:

**Additional information**: (Optional)

**Comment by Ecking03** [comment]
Ecking03
I was Banned on: 11/28/2020
I was banned by: Server
I was banned for saying N****R0
I would like to be unbanned because: My older brother was on my account and had typed that in on a name tag as a joke, and that resulted in me being banned. So if you could please give me another chance, I won't let my older brother on my account again.

**Comment by Caliteo** [comment]
Start over, be honest.

**Comment by Ecking03** [comment]
I would like to be unbanned because: I had noticied when joining this server that you were allowed to curse, so I took this as I could say any curse words. So I put the N word on a name tag as a joke, as I thought no one would see it, and it wouldn't cause any harm to anyone since I wasn't saying it in chat. But to my suprise, I was instantly perm banned. So I was hoping for a second chance, and I promise to never type that word again. (sorry for the dishonesty earlier)
